ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -700,20 +698,43 @@ else
                     // Response Code enum is in Helpers/SubmissionEvaluator
                     const messageArea = document.querySelector("#initialMessage");
                     let updateVal = "";
+                    let responseString = (responseData.completeResponse ?? "") + (responseData.freeformResponse ?? "");
+                    if (responseString.startsWith("Html.Raw")) { 
```

Comment:
Can you use RawHtmlHelper.Display() here instead?

Reply:
Makes sense, but please move this logic to a new helper function in RawHtmlHelper so that we don't lose this location if we ever change/expand the format. RawHtmlHelper.AsString perhaps?

[side note: I don't understand the ASP format at all lol]

Reply:
Can't do that either.  This data is coming from Javascript after the page is already loaded, so it has no concept of anything ASP.  The only time we can use ASP in Javascript is for creating static strings on page load.

Reply:
Understood, but please at least leave a note in RawHtmlHelper.cs pointing at this other usage.